examples/app-basic/src/module.ts

```diff
@@ -0,0 +1,12 @@
+import { AppPlugin } from '@grafana/data';
+import { App } from 'components/App';
+import { AppConfig } from 'components/AppConfig';
+
+export const plugin = new AppPlugin<{}>().setRootPage(App).addConfigPage({
+  title: 'Configuration',
+  icon: 'fa fa-cog',
+  // @ts-ignore - Would expect a Class component, however works absolutely fine with a functional one
+  // Implementation: https://github.com/grafana/grafana/blob/fd44c01675e54973370969dfb9e78f173aff7910/public/app/features/plugins/PluginPage.tsx#L157
+  body: AppConfig,
```

Comment:
We should probably update the typings in grafana to reflect this.

Reply:
Is it ok we tackle it in the following issue separately and update this example when the change has landed in Grafana?
https://github.com/grafana/grafana/issues/45791